lib/runcible/instance.rb

```diff
@@ -72,14 +84,6 @@ def accessible_class(class_object)
       #  (e.g. consumer_group)
       class_object.name.split('::').last.underscore
     end
-
-    def self.gather_classes(type)
```

Comment:
Would it be appropriate then to move all `self.` methods in the `class << self` block so that the two styles are not being mixed?